src/test/java/hudson/plugins/distfork/DistForkCommandTest.java

```diff
@@ -91,7 +107,7 @@ public void testAnonymousAccess() throws Exception {
         jr.jenkins.setAuthorizationStrategy(authz);
         
         String result = commandAndOutput("dist-fork", "-l", "master", "whoami");
-        assertThat(result, containsString("hudson.security.AccessDeniedException2: anonymous is missing the Slave/Build permission"));
+        assertThat(result, containsString(new AccessDeniedException2(Jenkins.ANONYMOUS, Computer.BUILD).getMessage()));
```

Comment:
now `Agent/Build`, and exception class name no longer printed (IIRC part of the general cleanup of CLI return codes)